lib/get-vue-sfc-script.js

```diff
@@ -22,7 +22,7 @@ module.exports.getVueSFCScript = (source, filename) => {
 			const { parse: parse2 } = require('@vue/component-compiler-utils');
 			const compiler = require('vue-template-compiler');
 
-			const { script } = parse2({ compiler, source, filename });
+			const { script } = parse2({ compiler, compilerParseOptions: { pad: 'space' }, source, filename });
```

Comment:
Do you have a link for some docs that led you to make this change or was it just trial and error?

Reply:
ref https://www.npmjs.com/package/vue-template-compiler

pad
pad is useful when you are piping the extracted content into other pre-processors, as you will get correct line numbers or character indices if there are any syntax errors.

with { pad: "line" }, the extracted content for each block will be prefixed with one newline for each line in the leading content from the original file to ensure that the line numbers align with the original file.
with { pad: "space" }, the extracted content for each block will be prefixed with one space for each character in the leading content from the original file to ensure that the character count remains the same as the original file.